OwenConferenciasApp/screens/ProfileUserLayout.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {View, Image, TouchableOpacity, Text, StyleSheet, Alert} from 'react-native';
```

Comment:
Here you are calling the Alert component and not using it. Let's call only the components we need. 🙏